docs/source-app/workflows/mount_cloud_object_store.rst

```diff
@@ -0,0 +1,134 @@
+:orphan:
+
+######################################################
+Mount Data From a Cloud Object Store to the Filesystem
+######################################################
+
+**Audience:** Users who want to read files stored in a Cloud Object Bucket in an app.
+
+******************************
+Mounting Public AWS S3 Buckets
+******************************
+
+=============================
+Configuring a Mount in a Work
+=============================
+
+To mount data from a cloud bucket to your app compute, initialize a ``Mount`` object with the source path and
+the absolute directory path where it should be mounted and pass it to the ``CloudCompute`` it should be mounted on.
+
+.. code:: python
+    :emphasize-lines: 9-12
+
+    import lightning as L
+    from lightning_app import CloudCompute
+    from lightning_app.storage import Mount
+
+    class Flow(L.LightningFlow):
+       def __init__(self):
+           super().__init__()
+           self.my_work = MyWorkClass(
+               cloud_compute=CloudCompute(
+                   mounts=Mount(
+                       source="s3://ryft-public-sample-data/esRedditJson/",
+                       mount_path="/content/esRedditJson/",
+                   ),
+               )
+           )
+
+       def run(self):
+           self.my_work.run()
+
+You can also pass multiple mounts to a single work by passing a ``List[Mount(...), ...]`` to the
```

Comment:
I would add notes here:

- Mounts are limited to 1M files.
- Mount objects should not be larger than 5 GB.
- When adding multiple mounts, each one should have a unique mount_path.